changelog/unreleased/fix-auth-allow-empty-cred-chain.md

```diff
@@ -0,0 +1,11 @@
+Bugfix: fix reva config of frontend service to avoid missleading error logs
+
+We set an empty Credentials chain for the frontend service now. In ocis all
+non-reva token authentication is handled by the proxy. This avoid irritating
+error messages about the missing 'auth-bearer' service.
```

Comment:
```suggestion
Bugfix: fix reva config of frontend service to avoid misleading error logs

We set an empty Credentials chain for the frontend service now. In ocis all
non-reva token authentication is handled by the proxy. This avoids irritating
error messages about the missing 'auth-bearer' service.
```